app/helpers/Emails/EmailHelper/EmailHelper.php

```diff
@@ -40,20 +46,28 @@ public function __construct(IMailer $mailer, array $params) {
     $this->footerUrl = Arrays::get($params, "footerUrl", "https://recodex.mff.cuni.cz");
     $this->siteName = Arrays::get($params, "siteName", "ReCodEx");
     $this->githubUrl = Arrays::get($params, "githubUrl", "https://github.com/ReCodEx");
+    $this->from = Arrays::get($params, "from", "ReCodEx <noreply@recodex.mff.cuni.cz>");
+    $this->subjectPrefix = Arrays::get($params, "subjectPrefix", "ReCodEx - ");
   }
 
   /**
    * Send an email with a nice template.
-   * @param string $from Sender of the email
+   * @param string|null $from Sender of the email
    * @param array $to Receivers of the email
    * @param string $subject Subject of the email
    * @param string $text Text of the message
    * @param array $bcc Blind copy receivers
    * @return bool If sending was successful or not
    */
-  public function send(string $from, array $to, string $subject, string $text, array $bcc = []) {
+  public function send(?string $from, array $to, string $subject, string $text, array $bcc = []) {
```

Comment:
Couldn't we make a separate method to avoid calling `->send(null, "someone@example.com", ...)`? I mean something like `sendFromDefaultAddress`.

Reply:
We definitely can